mongoose-movies/views/layout.hbs

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>{{title}}</title>
+    <link rel='stylesheet' href='/stylesheets/style.css' />
```

Comment:
Don't forget the meta tag for the viewport configuration, is in the cheatsheet. And very important